models.py

```diff
@@ -0,0 +1,29 @@
+# This encodes the different models
+
+def m1(expe,alphaG = .1,alphaN=.1,alphaC=.1,betaG=1,betaN=1):
+    """
+    This encodes RL model type 1
+    :param Expe: is the structure of the experiment through which M1 is run
```

Comment:
When not obvious, specify the expected types of your arguments. So e.g. `:param exp: Experiment instance describing the structure...`